ogzaf/demographics.py

```diff
@@ -27,6 +27,9 @@
 from ogcore import parameter_plots as pp
 from pandas_datareader import wb
 import matplotlib.pyplot as plt
+import requests
```

Comment:
I think this is unused now that you moved the `get_legacy_session()` functions to `utils.py`

Reply:
I'm not sure if we use requests somewhere else. I can remove it and see if something else breaks (we also access world bank data, for example)